src/InputFile/index.js.flow

```diff
@@ -13,7 +13,7 @@ export type Props = {|
   onChange?: (ev: SyntheticInputEvent<HTMLInputElement>) => void | Promise<any>,
   onFocus?: (ev: SyntheticInputEvent<HTMLInputElement>) => void | Promise<any>,
   onBlur?: (ev: SyntheticInputEvent<HTMLInputElement>) => void | Promise<any>,
-  onRemoveFile?: () => void,
+  onRemoveFile?: (ev: SyntheticEvent<HTMLButtonElement>) => void,
```

Comment:
```
onRemoveFile?: () => void | Promise<any>
```

I can't imagine situation when `ev` of button click is needed

Reply:
You're right, but we need to flow type the `ev` argument 😞

Reply:
if you don't pass in onRemoveFile fci argument `ev` you don't need it `onRemoveFile()`